scripts/release_info.py

```diff
@@ -411,6 +513,156 @@ def main():
+            changelog_content[2:2] = release_version_changelog
+            with python_utils.open_file(
+                CHANGELOG_FILEPATH, 'w') as changelog_file:
+                for line in changelog_content:
+                    changelog_file.write(line)
+            python_utils.PRINT('Updated Changelog!')
+
+            python_utils.PRINT('Updating Authors...')
+            start_index = release_summary_content.index(
+                '### New Authors:\n') + 1
+            end_index = release_summary_content.index(
+                '### Existing Authors:\n') - 1
+            new_authors = release_summary_content[start_index:end_index]
+            new_authors = [author.replace('* ', '') for author in new_authors]
+            update_sorted_file(
+                AUTHORS_FILEPATH, new_authors,
+                '# Please keep the list sorted alphabetically.\n')
+            python_utils.PRINT('Updated Authors!')
+
+            python_utils.PRINT('Updating Contributors...')
+            start_index = release_summary_content.index(
+                '### New Contributors:\n') + 1
+            end_index = release_summary_content.index(
+                '### Email C&P Blurbs about authors:\n') - 1
+            new_contributors = (
+                release_summary_content[start_index:end_index])
+            new_contributors = [
+                contributor.replace(
+                    '* ', '') for contributor in new_contributors]
+            update_sorted_file(
+                CONTRIBUTORS_FILEPATH, new_contributors,
+                '# Please keep the list sorted alphabetically.\n')
+            python_utils.PRINT('Updated Contributors!')
+
+            python_utils.PRINT('Updating Credits...')
+            new_credits = [
+                contributor.split('<')[0] for contributor in new_contributors]
+            new_credits.sort()
+            credit_dict = collections.defaultdict(list)
+            for credit in new_credits:
+                credit_dict[credit[0].upper()].append(
+                    '%s<li>%s</li>\n' % (LI_INDENT, credit))
+            with python_utils.open_file(
+                ABOUT_PAGE_FILEPATH, 'r') as about_page_file:
+                about_page_content = about_page_file.readlines()
+                for char in credit_dict:
+                    start_index = about_page_content.index(
+                        '%s<span>%s</span>\n' % (SPAN_INDENT, char)) + 2
+                    if char != 'Z':
+                        nxt_char = chr(ord(char) + 1)
+                        end_index = about_page_content.index(
+                            '%s<span>%s</span>\n' % (SPAN_INDENT, nxt_char)) - 2
+                    else:
+                        nxt_line = (
+                            '            <p translate="I18N_ABOUT_PAGE_CREDITS_'
+                            'TAB_TEXT_BOTTOM" translate-values="{listOfNames: '
+                            '$ctrl.listOfNames}">\n')
```

Comment:
Should we add a comment around this line of about page html file? 

Maybe we can create `user-list` directive which will only contain the list of names? (not sure about this)

Reply:
Added a comment.

I am sorry, I did not get the second part clearly. Are you suggesting to have a separate directive to only display the credits?